File: Beecrowd/1172/1172.cpp
#include <iostream>
using namespace std;

int main(){
        int x;
        int arr[10];
        for(int i = 0; i < 10; i++){
                cin >> x;
                if(x <= 0){ x = 1; }
                arr[i] = x;
                printf("X[%d] = %d\n", i, arr[i]);
        }

        return 0;
}

File: Beecrowd/1172/1172.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1172
arr = []
for i in range(0, 10):
    x = int(input())
    if x <= 0:
        x = 1
    arr.append(x)

for i in range(0, 10):
    print(f'X[{i}] = {arr[i]}')

